airflow/utils/helpers.py

```diff
@@ -150,19 +151,50 @@ def as_flattened_list(iterable):
 
 
 def chain(*tasks):
-    """
+    r"""
```

Comment:
What is this `r` for?

Reply:
> Both string and bytes literals may optionally be prefixed with a letter 'r' or 'R'; such strings are called raw strings and treat backslashes as literal characters

This makes the `\` in the ascii diagram be a backslash. I imagine otherwise it would complain about `\ ` being an invalid escape.

Reply:
Makes sense 👍 Thanks @ashb

Reply:
Thanks ash for the clarification